tests/providers/sftp/hooks/test_sftp.py

```diff
@@ -180,11 +178,11 @@ def test_no_host_key_check_default(self, get_connection):
 
     @mock.patch("airflow.providers.sftp.hooks.sftp.SFTPHook.get_connection")
     def test_no_host_key_check_enabled(self, get_connection):
-        connection = Connection(login="login", host="host", extra='{"no_host_key_check": false}')
+        connection = Connection(login="login", host="host", extra='{"no_host_key_check": true}')
 
         get_connection.return_value = connection
         hook = SFTPHook()
-        assert hook.no_host_key_check is False
+        assert hook.no_host_key_check is True
```

Comment:
Fix this test. 
It might be a typo because it had same implementations as `test_no_host_key_check_disabled` initially